discovery-infra/tests/conftest.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+import os
+
+class EnvVariable:
+    SSH_PUBLIC_KEY = os.environ.get('SSH_PUB_KEY').strip()
+    REMOTE_SERVICE_URL = os.environ.get('REMOTE_SERVICE_URL').strip()
+    PULL_SECRET = os.environ.get('PULL_SECRET').strip()
+    CLUSTER_NAME = os.environ.get('CLUSTER_NAME').strip()
+    OFFLINE_TOKEN = os.environ.get('OFFLINE_TOKEN').strip()
+    OPENSHIFT_VERSION = os.environ.get('OPENSHIFT_VERSION').strip()
+    BASE_DOMAIN = os.environ.get('BASE_DOMAIN').strip()
+    ISO_DOWNLOAD_PATH = os.environ.get('ISO').strip()
+    NUM_MASTERS = int(os.environ.get('NUM_MASTERS').strip())
+    NUM_WORKERS = int(os.environ.get('NUM_WORKERS').strip())
+    NUM_NODES = NUM_WORKERS + NUM_MASTERS
+    VIP_DHCP_ALLOCATION = os.environ.get('VIP_DHCP_ALLOCATION').strip() == 'True'
```

Comment:
`distutils.util.strtobool(os.environ.get('VIP_DHCP_ALLOCATION').strip())`

Reply:
util.strtobool returns 0/1 used: bool(util.strtobool(os.environ.get('VIP_DHCP_ALLOCATION'))) to match expected value in client